cohort-26-website-committee/class-website
# Nc thanks

# Description

added links and summary to most students on rows 1, 5, and 6
Fixes # (issue)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

Pull down my branch 
hs -o to start server 
test all links for the 3 rows 
check the modals for their description

# Checklist:
